presto-oracle/src/main/java/com/facebook/presto/plugin/oracle/OracleClientModule.java

```diff
@@ -49,6 +49,12 @@ public static ConnectionFactory connectionFactory(BaseJdbcConfig config, OracleC
             throws SQLException
     {
         Properties connectionProperties = new Properties();
+        if (oracleConfig.getTrustStorePath() != null) {
+            connectionProperties.setProperty("javax.net.ssl.trustStore", oracleConfig.getTrustStorePath());
```

Comment:
Use below imports instead,
import static oracle.jdbc.OracleConnection.CONNECTION_PROPERTY_THIN_JAVAX_NET_SSL_TRUSTSTORE;
import static oracle.jdbc.OracleConnection.CONNECTION_PROPERTY_THIN_JAVAX_NET_SSL_TRUSTSTOREPASSWORD;

Reply:
Updated the code accordingly.